include/crpropa/module/Output.h

```diff
@@ -63,27 +99,72 @@ class Output: public Module {
 	std::string OutputTypeName(OutputType outputtype);
 	const std::string outputName;
 	
+	/** Default constructor. Output contains all the information available.
+	 Units of energy and length are, by default, eV and Mpc.
+	 This can be changed with setEnergyScale and setLengthScale.
+	 */
 	Output();
-	Output(OutputType outputtype);
+	/** General constructor.
+	 Units of energy and length are, by default, eV and Mpc.
```

Comment:
Isn't the default energy unit EeV?

Reply:
True, this is a typo and I fixed it. However, I think it makes more sense to adopt eV as the default unit for 3.2, since we are covering ~GeV up to EeV with the same code.